scripts/create_test

```diff
@@ -257,7 +257,7 @@ OR
         args.no_batch = True
 
     if args.test_id is None:
-        args.test_id = CIME.utils.get_timestamp()
+        args.test_id = "%s.%s"%(CIME.utils.get_timestamp(), random.randrange(0,99,1))
```

Comment:
I like this idea but I'm worried that 0-99 is not enough entropy. I found this via google:
```
>>> import uuid
>>> print str(uuid.uuid4())[:8]
```
We could use :4 here. That would give us entropy of 16^4